projects/sbb-esta/angular-business/src/lib/contextmenu/contextmenu/contextmenu.component.scss

```diff
@@ -0,0 +1,94 @@
+@import '../../../styles/common/common';
```

Comment:
Just use 'common'. The reason for this is, that we use styleIncludePaths to use the appropriate includes for business vs public.